torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -316,24 +322,28 @@ bool FunctionSignature::parse(PyObject* args, PyObject* kwargs, PyObject* dst[],
     PyObject* obj = nullptr;
     if (arg_pos < nargs) {
       obj = PyTuple_GET_ITEM(args, arg_pos);
+      if (obj == Py_None && param.allow_none) {
```

Comment:
shouldn't we do something similar here for the kwarg check below?  Otherwise I can't pass Py_None as a kwarg.